catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -219,9 +221,8 @@
       new NitfHeaderAttribute(
           Security.RELEASABILITY,
           "FSREL",
-          header -> header.getFileSecurityMetadata().getReleaseInstructions(),
-          new SecurityAttributes().getAttributeDescriptor(Security.RELEASABILITY),
-          FILE_RELEASING_INSTRUCTIONS);
+          header -> handleReleasability(header.getFileSecurityMetadata().getReleaseInstructions()),
```

Comment:
❓ Looks like we dropped the last `FILE_RELEASING_INSTRUCTIONS`. Was that on purpose?

Reply:
Yes. Similar answer to `FILE_CLASSIFICATON_SECURITY_SYSTEM `